src/autoscaler/metricscollector/collector/app_streamer_test.go

```diff
@@ -93,7 +94,7 @@ var _ = Describe("AppStreamer", func() {
 				}))
 
 			})
-			Context("when save metrics to database fails", func() {
+			Context("when database fails", func() {
```

Comment:
nothing special, just want to correct the Grammar.  Changing to 'when saving to database fails'